internal/handler/join/handler.go

```diff
@@ -25,12 +26,14 @@ type handler struct{}
 
 func (h *handler) chatJoinRequest(ctx context.Context, bot *telego.Bot, request telego.ChatJoinRequest) {
 	log := logger.FromContext(ctx)
-	log.Infof(
-		"[JOIN REQUEST] username: %s, firstname: %s, id: %v",
-		request.From.Username,
-		request.From.FirstName,
-		request.From.ID,
-	)
+
+	log = log.With(slog.Group("user",
+		slog.String("username", request.From.Username),
+		slog.String("first_name", request.From.FirstName),
```

Comment:
I'm not sure if we really need first_name cos username should be enough to identify the user.